utils.js

```diff
@@ -0,0 +1,24 @@
+export function renderGoblin(goblinData) {
+    const goblinEl = document.createElement('div');
+    const faceEl = document.createElement('img');
+    const nameEl = document.createElement('p');
+    const hpEl = document.createElement('p');
+
+    goblinEl.classList.add('goblin');
+
+    nameEl.textContent = goblinData.name;
+    hpEl.id = `goblin-hp-${goblinData.id}`;
```

Comment:
To make this more legible against the background, you can add some css styling to this id (same with the hp and user data)